lib/topological_inventory/api/messaging.rb

```diff
@@ -0,0 +1,18 @@
+module TopologicalInventory
+  module Api
+    class Messaging
```

Comment:
it's a class there, but Module in Sources API, pls unify it

Reply:
Doen